delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/RemoteDiskStorageAccess.scala

```diff
@@ -1,14 +1,24 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote
 
+import akka.actor.ActorSystem
+import cats.syntax.all._
 import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageRejection.StorageNotAccessible
 import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageValue.RemoteDiskStorageValue
 import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.StorageAccess
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote.client.RemoteDiskStorageClient
 import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.AuthToken
 import monix.bio.IO
+import monix.execution.Scheduler
 
-object RemoteDiskStorageAccess extends StorageAccess {
+class RemoteDiskStorageAccess(implicit as: ActorSystem, sc: Scheduler) extends StorageAccess {
```

Comment:
We could create only one instance of this client and pass it along to the access, fetch and save operations so that they don't need to know about the ActorSystem or the Scheduler which are only related to the client.

Reply:
If you see the signatures, the `ActorSystem` is still needed for `S3DiskStorageAccess`. Scheduler is passed anyway to the `Storages` (notice I didn't have to change the current places where `evaluate` is being called.

So that seems to be the simplest solution.